src/server/server_family.cc

```diff
@@ -1069,6 +1070,52 @@ void ServerFamily::Client(CmdArgList args, ConnectionContext* cntx) {
   return (*cntx)->SendError(UnknownSubCmd(sub_cmd, "CLIENT"), kSyntaxErrType);
 }
 
+void ServerFamily::Cluster(CmdArgList args, ConnectionContext* cntx) {
+  ToUpper(&args[1]);
+  string_view sub_cmd = ArgS(args, 1);
+
+  if (dfly::is_emulated_cluster == false) {
+    return (*cntx)->SendError("CLUSTER commands demands the `cluster_mode` flag set to `emulated`");
+  }
+
+  if (sub_cmd == "HELP") {
+    string_view help_arr[] = {
+        "CLUSTER <subcommand> [<arg> [value] [opt] ...]. Subcommands are:",
+        "SLOTS",
+        "    Returns details about which cluster slots map to which instances.",
+        "HELP",
+        "    Prints this help.",
+    };
+    return (*cntx)->SendSimpleStrArr(help_arr, ABSL_ARRAYSIZE(help_arr));
+  }
+
+  if (sub_cmd == "SLOTS") {
+    (*cntx)->StartArray(1);    // Replying just one cluster
+    (*cntx)->StartArray(3);    // Should return a replica with master info?
+    (*cntx)->SendLong(0);      // start slot
+    (*cntx)->SendLong(16383);  // end slot
+    string cluster_announce_ip = GetFlag(FLAGS_cluster_announce_ip);
+    string_view ip_addr = cluster_announce_ip.empty()
+                              ? "127.0.0.1"  // TODO(lsvmello): where can I get this value?
+                              : cluster_announce_ip;
+    (*cntx)->StartArray(4);
+    (*cntx)->SendBulkString(ip_addr);
+    (*cntx)->SendLong(GetFlag(FLAGS_port));
+    (*cntx)->SendBulkString(master_id());
+    // TODO(lsvmello): Redis-py it nos using the following fields
```

Comment:
again, by checking against redis 6, i see that it actually returns 3 items and not 4 with the last one being master_id.
therefore I am totally fine to remove the last record.